modules/salmon/1.0/salmon.smk

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env snakemake
+
+
+##### ATTRIBUTION #####
+
+
+# Original Author:  Helena Winata
+# Module Author:    Helena Winata
+# Contributors:     N/A
+
+
+##### SETUP #####
+
+
+# Import package with useful functions for developing analysis modules
+import oncopipe as op
+
+# Setup module and store module-specific configuration in `CFG`
+# `CFG` is a shortcut to `config["lcr-modules"]["salmon"]`
+CFG = op.setup_module(
+    name = "salmon",
+    version = "1.0",
+    subdirectories = ["inputs", "salmon", "outputs"]
+)
+
+# Define rules to be run locally when using a compute cluster
+localrules:
+    _salmon_input_fastq,
+    _salmon_output,
+    _salmon_all,
+
+
+##### RULES #####
+
+
+# Symlinks the input files into the module results directory (under '00-inputs/')
+rule _salmon_input_fastq:
+    input:
+        fastq_1 = CFG["inputs"]["sample_fastq_1"],
+        fastq_2 = CFG["inputs"]["sample_fastq_2"]
+    output:
+        fastq_1 = CFG["dirs"]["inputs"] + "fastq/{seq_type}--{genome_build}/{sample_id}.R1.fastq",
```

Comment:
Aren't we planning on keeping fastq gzipped? My recent change to bam2fastq should output them as gzipped. Shouldn't the symlinks be named "fastq.gz"?

Reply:
Great

Reply:
I have updated it to indicate fastq compression